Add tip to run autotuner after host restart (#1419)
redpanda-data/documentation: docs/deploy/deployment-option/self-hosted/manual/production/production-deployment.mdx

sudo rpk redpanda mode production
```

> To tune the hardware, on each node, run the autotuner:

Reviewer: Rewrite with one less comma:
```suggestion
To tune the hardware, run the autotuner on each node:
```

```bash
sudo rpk redpanda tune all
```

:::tip

> Run the autotuner every time you restart a Redpanda node's host, since a restart resets some tuning properties.

Reviewer 2: Hm, I need to figure out which one we should run; `systectl start redpanda-tuner` vs `rpk redpanda tune all`. The former actually runs the latter.

Reviewer 3: the systemd service `redpanda-tuner` will run every time the system is booted so that should be the recommended way to have `rpk redpanda tune all` run on boot on linux distros that have systemd.

Reviewer 2: Thanks, @andrewhsu. Let me clarify: 

> so that should be the recommended way to have rpk redpanda tune all run on boot on linux distros that have systemd

`redpanda-tuner` actually invokes `rpk redpanda tune all` behind the scenes, right? On the system is booted, why do we need to run the rpk command even though `redpanda-tuner` runs?

Reviewer 3: > redpanda-tuner actually invokes rpk redpanda tune all behind the scenes, right?

yes, what i meant is the changes in this PR should be updated to guide the user to use `systemctl` only since it will manage start/stop and also start on boot.

on rhel distros, after the rpm package is installed, the `redpanda-tuner` service is not started nor is it enabled to start on next boot. so the user should:
```
sudo systemctl start redpanda-tuner # this will run `rpk redpanda tune all`
```
and then
```
sudo systemctl enable redpanda-tuner # this will make systemd start the service on bootup
```

ubuntu distros will automatically have the service enabled after installing with apt so they just need to run:
```
sudo systemctl start redpanda-tuner
```

Reviewer 3: or as follow-up PR

Author: @andrewhsu @daisukebe - updating this PR for this comment

> Run the autotuner after every restart of a Redpanda node's host, as some tuned properties are reset when a node's host VM or machine is restarted.

Reviewer: ```suggestion
Run the autotuner every time you restart a Redpanda node's host, since a restart resets some tuning properties.
```
